app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -595,7 +595,8 @@ class LuciBuildService {
     await pubsub.publish('scheduler-requests', request);
 
     task.attempts = (task.attempts ?? 0) + 1;
-    task.status = Task.statusNew;
+    // Mark task as in progress to ensure it isn't scheduled over
+    task.status = Task.statusInProgress;
```

Comment:
Is this to prevent rescheduling? Or starting it over?

Reply:
Prevent duplicate re-scheduling from backfiller.